src/Plugin.php

```diff
@@ -257,7 +257,7 @@ public static function editions(): array
     /**
      * @inheritDoc
      */
-    public string $schemaVersion = '5.2.0.5';
```

Comment:
The first 3 numbers here should correspond with the plugin version it will be released as. So `5.2.7.0`.